lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
Ok but I guess its still not clear to me why registering for location updates would fail if `Looper.loop()` has already been called for a background thread.

I'm assuming `Looper.loop()` gets called for the main thread as part of app startup and definitely before we would register for location updates and in that scenario it works.